api/python/quilt3/packages.py

```diff
@@ -793,6 +806,11 @@ def build(self, name=None, registry=None, message=None):
         Returns:
             The top hash as a string.
         """
+        return self._build(name=name, registry=registry, message=message)
+
+
+    def _build(self, name=None, registry=None, message=None):
+        """   Telemetry-free version of build()   """
```

Comment:
It's a bit weird to mention telemetry in every function... I would remove the comments from the private functions.

Reply:
Agreed